### DISCUSSION TITLE
Como incluir o .NET Core no Visual Studio Code?

### DISCUSSION THREAD

**Discussion by laisrmesquita** [balta-io/forum#713] [Tira Dúvidas] [answered]
Curso: Fundamentos do C#
Aula: .NET Debug https://balta.io/player/assistir/5ef614da-366d-6156-d049-694800000000/ec5c34a0-6c58-423f-bec0-000028010408

No minuto 9:13 dessa aula o professor nos ensina a criar a pasta vscode. Porém ele mostra essa opção .NET Core e eu não encontro a opção no meu Visual Studio Code
Veja como aparece na aula:
![image](https://github.com/user-attachments/assets/0adae972-c207-4913-a730-e38713dca48b)

E como aparece pra mim:
![image](https://github.com/user-attachments/assets/620a2293-2ff4-4e65-8933-29fcf7bd4103)
Podem me ajudar com qual a opção correta, por favor?

Ou me ensinar a incluir o NET Core no meu programa.

**Comment by andrebaltieri** [comment, marked as answer]
É que o nome mudou! Você deve estar na versão mais recente... pode usar a primeira opção @laisrmesquita

**Comment by laisrmesquita** [comment]
@andrebaltieri bom dia, tudo bem?
Eu não sei o que eu fiz de errado mas agora quando eu clico em "run and debug" e depois em "create a launch.json file" as opções estão diferentes, não aparecem mais as do print
![image](https://github.com/user-attachments/assets/b7201b86-0591-420e-b9f4-6110fa916638)

Também não apareceu pra mim essa mensagem que você mostrou no vídeo de Estrutura do APP:
![image](https://github.com/user-attachments/assets/ada82659-d244-4baf-81d1-ec188575ac6e)

**Reply by andrebaltieri** [reply]
Tá vendo essa mensagem no canto inferior direito da tela, aperta em `Yes` que ele vai instalar o que precisa!!
Ai vai funcionar!

**Reply by laisrmesquita** [reply]
a mensagem não apareceu pra mim :/

**Reply by andrebaltieri** [reply]
Consegue verificar se a extensão do C# está instalada?
![Screenshot 2024-12-12 at 08 09 05](https://github.com/user-attachments/assets/9ea91512-01e1-44f6-9ae8-28ddbffb6d9a)

**Reply by laisrmesquita** [reply]
oi, bom dia! eu desistalei o visual studio, instalei de novo e fiz todo o processo novamente. agora a mensagem apareceu, certinho aqui! obrigada

**Reply by andrebaltieri** [reply]
Que bom @laisrmesquita, obrigado por compartilhar a solução!
